ui/views.py

```diff
@@ -21,6 +21,13 @@ def homepage(request):
             start_time__range=get_current_week()
         ).values_list("assigned_movie", flat=True)
     )
+
+    playing_movies_next_week = Movie.objects.filter(
+        pk__in=PlayingTime.objects.filter(
+            start_time__range=get_next_week()
+        ).values_list("assigned_movie", flat=True)
+    )
+    print(playing_movies_next_week)
     filter = PlayingTimeFilter(request.GET, queryset=PlayingTime.objects.all())
     paginator = Paginator(playing_movies, 10)
```

Comment:
Shouldn't the paginator iterate `playing_movies_next_week`?

Reply:
playing_movies_next_week are in a carousel.